EXOFiddlerInspector/CheckForAppUpdate.cs

```diff
@@ -74,36 +64,47 @@ public void CheckForUpdate()
             }
 
             Version applicationVersion = System.Reflection.Assembly.GetExecutingAssembly().GetName().Version;
+
             /// <remarks>
             /// Update available.
             /// </remarks>
             /// 
-
             if (applicationVersion.CompareTo(newVersion) < 0)
             {
 
                 Debug.WriteLine($"EXCHANGE ONLINE EXTENSION: {DateTime.Now}: CheckForAppUpdate.cs : Update Available.");
-                FiddlerApplication.Prefs.SetStringPref("extensions.EXOFiddlerExtension.MenuTitle", "Exchange Online (Update Available)");
+                //FiddlerApplication.Prefs.SetStringPref("extensions.EXOFiddlerExtension.MenuTitle", "Exchange Online (Update Available)");
 
                 /// <remarks>
                 /// Refresh the value of ManualCheckForUpdate and respond with feedback if needed.
                 /// </remarks>
 
-                Boolean ManualCheckForUpdateFeedback = FiddlerApplication.Prefs.GetBoolPref("extensions.EXOFiddlerExtension.ManualCheckForUpdate", false);
+                //Boolean ManualCheckForUpdateFeedback = FiddlerApplication.Prefs.GetBoolPref("extensions.EXOFiddlerExtension.ManualCheckForUpdate", false);
+
+                if (applicationVersion.Build >= 1000)
+                {
+                    FiddlerApplication.Prefs.SetStringPref("extensions.EXOFiddlerExtension.UpdateMessage", $"Update Available{Environment.NewLine}----------------" +
```

Comment:
// If we are using Preferenes to manage app preferences, we should continue to do so and not have disjointed approach. 
// Suggest we add a new property to Preferences to handle all the FiddlerApplication.Prefs calls in this class.